src/grid/grid.pinning.spec.ts

```diff
@@ -584,6 +584,24 @@ describe("IgxGrid - Column Pinning ", () => {
         expect(grid.unpinnedColumns.length).toEqual(unpinnedColumnsLength);
         expect(result).toBe(false);
     });
+
+    it("should minimal visible unpinned area width be 20% of the grid width", () => {
```

Comment:
Can you make the test name a coherent sentence?

Currently  the test name is:
IgxGrid - Column Pinning should minimal visible unpinned area width be 20% of the grid width.

A more coherent name would be something like:
igxGrid – Column Pinning should reject pinning a column if the unpinned area width becomes less than 20% of the grid width